src/patterns/wmf-accordion.scss

```diff
@@ -159,21 +159,11 @@ $primary-nav-mobile-height: 62px;
 	.wmf-accordion-item {
 		background: #fff;
 		border: 1px solid #aaa;
-		border-left: 0;
+		border-left: 0.5em solid #ff6801;
```

Comment:
Should this be driven by the selected color?

Reply:
This is a fallback to the default colour that was being set elsewhere. If the colour is selected, that will take precedence over this.